property.html

```diff
@@ -276,6 +285,37 @@ <h1>Form values, set</h1>
 				10
 			</td>
 		</tr>
+		<tr title="&lt;select multiple> with one value">
```

Comment:
In general, it's good to decouple features we're testing. Here you are testing two things: whether `select[multiple]` works with 1 value, and whether it works with `mv-value`. I'd prefer adding two tests that test this separately.